src/style/modal/user-modal.less

```diff
@@ -27,4 +27,17 @@
   .panel-participant {
     padding: 0;
   }
+
+  .modal__header {
+    margin: 16px;
+  }
+
+  .modal__body {
+    padding: 16px;
+
+    .classified-bar {
+      width: calc(100% + 32px);
+      margin-top: 12px;
+    }
+  }
```

Comment:
Is there anyway we could colocate those styles next to the targeted component in a `Component.styles.ts` file?